pages/education/apply-for-education-benefits/application/1995-STEM.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Veteran request for change
+entryname: 1995-edu-benefits
```

Comment:
This entry name needs to be different than the regular 1995 form, assuming you're making a copy of it.

Also, vets-website will automatically generate a basic entry page for you in dev and staging (this is a change in the past few months), so you don't necessarily have to create this page now (though you will need it eventually).